fastapi_utils/crud_base.py

```diff
@@ -0,0 +1,160 @@
+from decimal import Decimal
```

Comment:
Any chance we could put this file (`crud_base.py`) and (`base_route.py`) into a `crud` subfolder/package? It could be `fastapi_utils.crud.base` and `fastapi_utils.crud.route`.

Reply:
Sure